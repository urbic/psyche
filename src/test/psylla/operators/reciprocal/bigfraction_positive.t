2. sqrt torational dup mul reciprocal ?
