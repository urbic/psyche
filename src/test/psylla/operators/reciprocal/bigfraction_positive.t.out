20282409603651670423947251286016:40564819207303346393761349247529
